docs/getting-started/install/debian.md

```diff
@@ -3,13 +3,13 @@
 As of 0.3.18, dokku defaults to being installed via debian package. While certain hosts may require extra work to get running, you may optionally wish to automate the installation of dokku without the use of our `bootstrap.sh` bash script. The following are the steps run by said script:
 
 ```shell
-curl --silent https://get.docker.io/gpg 2> /dev/null | apt-key add - 2>&1 >/dev/null
-curl --silent https://packagecloud.io/gpg.key 2> /dev/null | apt-key add - 2>&1 >/dev/null
+curl -sL https://get.docker.io/gpg 2> /dev/null | sudo apt-key add - 2>&1 >/dev/null
+curl -sL https://packagecloud.io/gpg.key 2> /dev/null | sudo apt-key add - 2>&1 >/dev/null
 
-echo "deb http://get.docker.io/ubuntu docker main" > /etc/apt/sources.list.d/docker.list
-echo "deb https://packagecloud.io/dokku/dokku/ubuntu/ trusty main" > /etc/apt/sources.list.d/dokku.list
+sudo echo "deb http://get.docker.io/ubuntu docker main" > /etc/apt/sources.list.d/docker.list
```

Comment:
This specific line doesn't work, since only echo is being sudo'd. Needs to be wrapped in "sh -c" so it is all sudo, or piped to `sudo tee /etc/apt/sources.list.d/docker.list`

Also, i'm not sure I'm not sure this is the behavior one might expect, a script escalating itself via sudo.